rotkehlchen/db/evmtx.py

```diff
@@ -533,6 +568,20 @@ def delete_transactions(
             [(f'{EXTRAINTERNALTXPREFIX}_{chain_id.value}_%_{tx_hash.hex()}',) for tx_hash in tx_hashes],  # noqa: E501
         )
 
+        # delete orphan topic ids
+        write_cursor.execute(
```

Comment:
what if this raises integrity error? I mean it should not but .. should it be handled? Or if it somehow does it's a bad error in the logic so let it raise?